packages/expo-updates/ios/EXUpdates/Update/RawManifest/EXUpdatesNewRawManifest.m

```diff
@@ -0,0 +1,44 @@
+//  Copyright © 2021 650 Industries. All rights reserved.
+
+#import <EXUpdates/EXUpdatesNewRawManifest.h>
+
+@implementation EXUpdatesNewRawManifest
```

Comment:
should we have `nullthrows`-like checks in here?

Reply:
Yep, definitely. To minimize the scope of this PR, I opted to keep the null checking behavior at the callsites though and just annotate the methods with nullable/nonnull for now to help us determine what things may need to be nullable. In a follow-up I will be moving the nullthrows checks to these methods to maintain consistency with android.

On a side-note though I've already seen some discrepancies in certain fields between android and iOS, so those'll need to be addressed as well. Maybe we'll even be able to somehow share code across platforms.